staff/marti-malek/e-commerce-template/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Template</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/" crossorigin="anonymous">
+</head>
+<body>
+    <header>
+            <i id="logo" class="fas fa-toilet"></i>
+        <input type="text" placeholder="Search something...">
+        <button class="button2"><i class="fas fa-search"></i></button>
+        <div class="dropdown">
```

Comment:
Try to use the `nav` tag instead of the `div`here, will be more semantic

Reply:
Done! Thank you!